tests/annealing_box_unittests.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import numpy as np
+from syllabus.tests import create_cartpole_env
+from syllabus.curricula import AnnealingBoxCurriculum
+
+class TestAnnealingBoxCurriculum(unittest.TestCase):
```

Comment:
Really nice test! I didn't even know that numpy.testing existed. Please also add test cases for negative start_values/end_values, and reverse annealing (e.g. 0.7 -> 0.1 and 0.7 -> -0.1).